src/wallet/components/WalletDropdownDisconnect.test.tsx

```diff
@@ -1,23 +1,23 @@
 import { fireEvent, render, screen } from '@testing-library/react';
 import { describe, expect, it, vi } from 'vitest';
-import { useConnect, useDisconnect } from 'wagmi';
+import { type Connector, useDisconnect } from 'wagmi';
```

Comment:
nit, let's have `types` always on a different line of import so it will be
```
import { useDisconnect } from 'wagmi';
import type { Connector } from 'wagmi';
```